examples/plot_missing_values.py

```diff
@@ -9,16 +9,23 @@
 cross-validation.  Sometimes dropping rows or using marker values is
 more effective.
 
-Missing values can be replaced by the mean, the median or the most frequent
+Imputer:
```

Comment:
I'd also be tempted to just drop this detail that is in the user guide. The example's job is to show it in action not to explain the available usages.

Reply:
That's up to you, of course. I just added the small note for KNNImputer since one already existed for the default Imputer. Also, since the explanation is fairly short might be worth leaving in case user needs a quick description? Anyway, I am good either way.